req.txt

```diff
@@ -0,0 +1,12 @@
+aniso8601==8.0.0
+click==7.1.2
+Flask==1.1.2
+Flask-RESTful==0.3.8
```

Comment:
Where is Flask API?

Reply:
i didn't use it its commented in main.py